app/views/decidim/budgets/admin/projects/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= render partial: "decidim/shared/component_announcement" %>
```

Comment:
I think this file shouldn't be in the admin folder but in `app/views/decidim/budgets/projects/index.html.erb`

Reply:
Ok thx